x/concentrated-liquidity/position.go

```diff
@@ -490,9 +502,6 @@ func (k Keeper) fungifyChargedPosition(ctx sdk.Context, owner sdk.AccAddress, po
 				if err != nil {
 					return 0, err
 				}
-
-				// Delete the accumulator position from state.
-				uptimeAccum.DeletePosition(oldPositionName)
```

Comment:
Note: this is bug number 1 in the description where it wouldn't correctly decrement global accum shares.